OutaTime/OutaTime/TimeCurcuitViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  TimeCurcuitViewController.swift
+//  OutaTime
+//
+//  Created by Chris Gonzales on 1/29/20.
+//  Copyright © 2020 Chris Gonzales. All rights reserved.
+//
+
+import UIKit
```

Comment:
Love this file! It is very clean and organized. You have your variables up top. Then your Outlets. And then your functions. Very very good. Keep up this great clean structure!